config/locales/en.yml

```diff
@@ -991,11 +991,15 @@ en:
         email_form:
           label: Send an email
           submit: Send
+        email_opted_out: Client has opted out of emails.
         no_email_address: Client has not provided an email address.
+        no_email_and_opt_out: Client has not provided an email address and opted out of emails.
+        no_sms_and_opt_out: Client has not provided an SMS phone number and opted out of text messages.
         no_sms_number: Client has not provided an SMS phone number.
-        no_way_to_communicate: Client has not provided contact info for digital communication.
+        no_way_to_communicate: Client has not provided contact info for digital communication and/or has opted-out.
```

Comment:
maybe i'm misremembering how gyr works but isn't it maybe more accurate to say that they never opted in? or is there a way to opt out that i'm forgetting?